ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -641,4 +639,88 @@ def check_apps_running_on_selected_cluster(
         else:
             log.error(f"App {app} not found on cluster {cluster_name} on DR dashboard")
             return False
-    return True
+
+
+def verify_application_not_present_in_ui(acm_obj, workload_to_check, timeout=60):
+    """
+    Verify if application is not present in UI
+
+    Args:
+        acm_obj (AcmAddClusters): ACM Page Navigator Class
+        workload_to_check (str): Specify the workload to check,
+        timeout (int): timeout to wait for certain elements to be found on the ACM UI
+
+    Returns:
+        bool: True if the application is not present, false otherwise
+
+    """
+    if workload_to_check:
```

Comment:
This condition will always return True based upon the current param in function `if workload_to_check`

Reply:
@Shrivaibavi ^^

Reply:
Now workload_to_check is a list, hence it will return false if its empty

Reply:
@Shrivaibavi Aren't you returning True in line 684 if the list is empty?

Reply:
Workload_to_check becomes a necessary argument, if its empty ideally it wont enter the loop. I hope this answer helps. 
And sorry if my above answer is confusing.

Reply:
I am just asking what would it return if the list is empty?

Reply:
Nothing... Ideally it shouldn't be an empty list. Validating if its an empty list or contains element and raising exception wouldn't make much difference as workloads_to_check had been added as necessary param IMO.

Reply:
No, ideally we should raise an exception or fail the test if the list is empty if what I feel.